backend/project/__init__.py

```diff
@@ -0,0 +1,34 @@
+#### BACKEND PROJECT PACKAGE INITIALIZATION ####
+
+
+from flask import Flask
+from flask_bcrypt import Bcrypt
+import mysql.connector as mariadb
+import os
+
+
+# Create instance of Flask application
+app = Flask(__name__, template_folder=os.path.abspath('../frontend'))
+# Apparently setting `SECRET_KEY` helps against XSS
+app.config['SECRET_KEY'] = 'a5cd2a20083251fa893622027c734d43'
+
+# Create usable instance of encryptor
+bcrypt = Bcrypt(app)
+
+# Establish database connection
+dbconnection = mariadb.connect(user='prod', password='changeme', database='team_pass', host='98.218.4.51')
```

Comment:
Can we use an environment variable for this password and the `SECRET_KEY`? One option is to use [python-dotenv](https://pypi.org/project/python-dotenv/), which allows us to specify a file like `production.env` which can be added to the `.gitignore`. We'd do something like:
`production.env`
```bash
DB_PASSWORD="blah"
FLASK_SECRET_KEY="supersecretmsg"
```
and in `__init__.py`
```python
app.config['SECRET_KEY'] = os.getenv('FLASK_SECRET_KEY', '<whatever the development default secret key is>')

password = os.getenv('DB_PASSWORD', '<whatever the development default db password is>')
dbconnection = mariadb.connect(user='prod', password=password, database='team_pass', host='98.218.4.51')
```